test/corpus/todo.txt

```diff
@@ -98,8 +98,28 @@ end
 
 (expressions)
 
+===============================
+pseudo-methods with macro nodes
+:error
+===============================
+a.is_a?({{type}})
```

Comment:
I think this is only allowed inside a macro context (macro def or `{% begin %}{% end %}`, which should already be supported by what you have

Reply:
The purpose of this todo is improving syntax highlighting for `is_a?` and other pseudomethods. Right now, `is_a?` isn't highlighted properly inside a macro body:
![image](https://github.com/user-attachments/assets/92af8a93-7360-4f84-8977-259cf6dd9c79)
Should be an easy followup, I'm just trying to keep the scope of this PR from growing any further :slightly_smiling_face: